OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -107,10 +107,14 @@ void PlayAttack(Actor self, WPos attackPosition, List<Player> affectedPlayers)
 			withSpriteBody.PlayCustomAnimation(self, sandworm.Info.MouthSequence);
 			Game.Sound.Play(swallow.Info.WormAttackSound, self.CenterPosition);
 
+			if (affectedPlayers.Count == 0)
+				return;
+
 			Game.RunAfterDelay(1000, () =>
 			{
-				foreach (var affectedPlayer in affectedPlayers)
-					NotifyPlayer(affectedPlayer, attackPosition);
+				if (Game.IsCurrentWorld(affectedPlayers[0].World))
```

Comment:
`Game.OrderManager` is not available from mod code. `self.World` would be a more readable choice than `affectedPlayers[0].World`